### PR TITLE
Feature/lh 25 python user tasks

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**
Working commit with scheduling of User Tasks working. Still needs:
* Automatic reassignment
* Schedule Reminder Tasks
* Add Notes to User Task

### CODE REVIEW

File: sdk-python/littlehorse/workflow.py [hunk 1]
```diff
@@ -634,6 +639,26 @@ def execute(self, task_name: str, *args: Any) -> NodeOutput:
         node_name = self.add_node(task_name, task_node)
         return NodeOutput(node_name)
 
+    def assign_user_task(
```
Comment (sauljabin) [inline sdk-python/littlehorse/workflow.py:642]: Some methods are missing right? and an example

**Review by sauljabin — APPROVED** [review]
code looks good, also we have talked already about the optimization you did

File: sdk-python/littlehorse/workflow.py (hunk 1 above)
Comment (coltmcnealy-lh) [reply sdk-python/littlehorse/workflow.py:642]: Yes and yes. Still in draft...

File: sdk-python/littlehorse/workflow.py [hunk 2]
```diff
@@ -494,6 +503,48 @@ def __str__(self) -> str:
         return to_json(self.compile())
 
 
+class LHFormatString:
```
Comment (sauljabin) [inline sdk-python/littlehorse/workflow.py:506]: this class already existed i guess

Comment (sauljabin) [reply sdk-python/littlehorse/workflow.py:506]:
```
class FormatString:
    def __init__(self, format: str, *args: Any) -> None:
        """Generates a FormatString object that can be understood by the ThreadBuilder.

        Args:
            format (str): String format with variables with curly brackets {}.
            *args (Any): Arguments.

        Returns:
            FormatString: A FormatString.
        """
        self.format = format
        self.args = args
```

**Pull request merged**